core/field_angle.ts

```diff
@@ -69,49 +68,38 @@ export class FieldAngle extends FieldInput<number> {
    * Whether the angle should increase as the angle picker is moved clockwise
    * (true) or counterclockwise (false).
    */
-  private clockwise_ = FieldAngle.CLOCKWISE;
+  private clockwise = FieldAngle.CLOCKWISE;
 
   /**
    * The offset of zero degrees (and all other angles).
    */
-  private offset_ = FieldAngle.OFFSET;
+  private offset = FieldAngle.OFFSET;
 
   /**
    * The maximum angle to allow before wrapping.
    */
-  private wrap_ = FieldAngle.WRAP;
+  private wrap = FieldAngle.WRAP;
 
   /**
    * The amount to round angles to when using a mouse or keyboard nav input.
    */
-  private round_ = FieldAngle.ROUND;
+  private round = FieldAngle.ROUND;
 
-  /** The angle picker's SVG element. */
-  private editor_: SVGSVGElement|null = null;
-
-  /** The angle picker's gauge path depending on the value. */
-  gauge_: SVGPathElement|null = null;
+  /**
+   * Array holding info needed to unbind events.
+   * Used for disposing.
+   * Ex: [[node, name, func], [node, name, func]].
+   */
+  private boundEvents: browserEvents.Data[] = [];
 
   /** The angle picker's line drawn representing the value's angle. */
-  line_: SVGLineElement|null = null;
-
-  /** The degree symbol for this field. */
-  protected symbol_: SVGTSpanElement|null = null;
+  line: SVGLineElement|null = null;
 
-  /** Wrapper click event data. */
-  private clickWrapper_: browserEvents.Data|null = null;
-
-  /** Surface click event data. */
-  private clickSurfaceWrapper_: browserEvents.Data|null = null;
-
-  /** Surface mouse move event data. */
-  private moveSurfaceWrapper_: browserEvents.Data|null = null;
+  /** The angle picker's gauge path depending on the value. */
+  gauge: SVGPathElement|null = null;
 
-  /**
-   * Serializable fields are saved by the serializer, non-serializable fields
-   * are not. Editable fields should also be serializable.
-   */
-  override SERIALIZABLE = true;
+  /** The degree symbol for this field. */
+  protected symbol: SVGTSpanElement|null = null;
```

Comment:
Protected properties are part of the public API :/ so the underscore should be re-added for now.